src/jira/client/axios.ts

```diff
@@ -147,13 +147,16 @@ const instrumentRequest = (response) => {
 const instrumentFailedRequest = (baseURL: string, logger: Logger) => {
 	return async (error: AxiosError) => {
 		instrumentRequest(error?.response);
-		if (error.response?.status === 503) {
+		if (error.response?.status === 503 || error.response?.status === 405) {
 			try {
 				await axios.get("/status", { baseURL });
 			} catch (e) {
 				if (e.response.status === 503) {
 					logger.info(`503 from Jira: Jira instance '${baseURL}' has been deactivated, is suspended or does not exist. Returning 404 to our application.`);
 					error.response.status = 404;
+				} else if (e.response.status === 302) {
```

Comment:
... and this one too...

Reply:
but probably not a big deal cause this is error handling

Reply:
Yeah I think it's ok because I'm just adding more logging to an error that is already being thrown. If these conditions aren't met, we will reject it anyways on line 163.